Satellite idm auth support 12891 (#1915)
theforeman/foreman-documentation: guides/common/modules/proc_using-IdM-credentials-to-log-in-to-the-Project-Web-UI-with-a-Firefox-browser.adoc

[id="Using_IdM_credentials_to_log_in_to_the_{Project}_Web_UI-with-a-Firefox-browser_{context}"]
= Using IdM credentials to log in to the {Project} Web UI with a Firefox browser

> This section describes how to use the Firefox browser to log in to your {Project} Web UI with your Identity Management (IdM) login + password.

Reviewer: Same as above.

.Prerequisites

> * Your system administrator has enrolled your {Project} server into IdM and configured the server to use IdM for authentication. For more information, see {InstallingServerDocURL}Configuring_External_User_Groups_{project-context}#Using_FreeIPA_admin[Using Red Hat Identity Management] in _{InstallingServerDocTitle}_.

Reviewer: same as above.

* An IdM server is running and reachable.

> * The host on which you are using a Firefox browser to log in to the {Project} Web UI is an IdM client.

Reviewer: ```suggestion
* The host on which you are using a Firefox browser to log in to the {ProjectWebUI} is an IdM client.
```

* You have a valid IdM login and password.

> * [Recommended] You are using the latest version of the Firefox browser.

Reviewer: Having the "[Recommended]" prefix is unusual. Maybe reword to "{Team} recommends using the latest stable Firefox browser."?

Author: Thank you, updated now.

* Your Firefox browser is configured for Single Sign-On (SSO).

.Procedure
. Open a terminal window.
. Obtain the Kerberos ticket granting ticket (TGT) for yourself using your IdM credentials:
+
[options="nowrap", subs="+quotes,verbatim,attributes"]
----

> kinit _idm_user_

Reviewer: ```suggestion
$ kinit _idm_user_
```

Author: Corrected in the latest commits

Password for idm_user@_EXAMPLE.COM_:
----

> . In your browser address bar, enter the URL of your {Project} Server.

Reviewer: ```suggestion
. In your browser address bar, enter the URL of your {ProjectServer}.
```

Author: Corrected in the latest commits

+
You are logged in automatically.

> 

Reviewer: ```suggestion
```

[NOTE]
====

> Alternatively, you can skip the first two steps and enter your login + password in the fields displayed on the {Project} Web UI. This is also the only option if the host from which you are accessing the {Project} Web UI is not an IdM client.

Reviewer: ```suggestion
Alternatively, you can skip the first two steps and enter your login and password in the fields displayed on the {ProjectWebUI}.
This is also the only option if the host from which you are accessing the {ProjectWebUI} is not an IdM client.
```

Author: Corrected in the latest commits

> [id="Using_IdM_credentials_to_log_in_to_the_{Project}_Web_UI-with-a-Firefox-browser_{context}"]

Reviewer: same as above.

= Using IdM credentials to log in to the {ProjectWebUI} with a Firefox browser

This section describes how to use the Firefox browser to log in to your {ProjectWebUI} with your {FreeIPA} (IdM) login and password.

.Prerequisites

> * Your system administrator has enrolled your {ProjectServer} into IdM and configured the server to use IdM for authentication.

Reviewer: ```suggestion
* You have enrolled your {ProjectServer} into IdM and configured the server to use IdM for authentication.
```

> For more information, see {InstallingServerDocURL}Configuring_External_User_Groups_{project-context}#Using_FreeIPA_admin[Using {FreeIPA}] in _{InstallingServerDocTitle}_.

Reviewer: ```suggestion
For more information, see {InstallingServerDocURL}Configuring_External_User_Groups_{project-context}Using_FreeIPA_admin[Using {FreeIPA}] in _{InstallingServerDocTitle}_.
```

Author: Same as above (replace Using_FreeIPA_admin with Using_FreeIPA_{project-context})

* An IdM server is running and reachable.
* The host on which you are using a Firefox browser to log in to the {ProjectWebUI} is an IdM client.
* You have a valid IdM login and password.
* {Team} recommends using the latest stable Firefox browser.
* Your Firefox browser is configured for Single Sign-On (SSO).

.Procedure

> . Open a terminal window.

Reviewer: Same as above.

Reviewer: We try to match file name, anchor, and title as closely as possible; I am unsure how to handle `{ProjectWebUI}` in a file name, so either we go with "WebUI" which is the upstream terminology, or you keep it as is. Both is valid.
